packages/zowe-explorer/src/webviews/src/table-view/ActionsBar.tsx

```diff
@@ -1,66 +1,174 @@
-import { Ref } from "preact/hooks";
+import { Dispatch, Ref, useState } from "preact/hooks";
 import type { Table } from "@zowe/zowe-explorer-api";
-import { VSCodeButton } from "@vscode/webview-ui-toolkit/react";
+import { VSCodeButton, VSCodeTextField } from "@vscode/webview-ui-toolkit/react";
 import { GridApi } from "ag-grid-community";
+import { wrapFn } from "./types";
+import { FocusableItem, Menu, MenuGroup, MenuItem } from "@szhsin/react-menu";
+import "@szhsin/react-menu/dist/index.css";
 
 export const ActionsBar = ({
   actions,
   gridRef,
   itemCount,
+  selectionCount,
+  title,
+  columns,
+  visibleColumns,
+  setVisibleColumns,
   vscodeApi,
 }: {
   actions: Table.Action[];
   gridRef: Ref<any>;
   itemCount: number;
+  selectionCount: number;
+  title: string;
+  columns: string[];
+  visibleColumns: string[];
+  setVisibleColumns: Dispatch<string[]>;
   vscodeApi: any;
```

Comment:
Curious if we should define maybe a local interface for this 😋 
That may help clean up a bit the function signature 😋

Reply:
That's a good idea, I've separated it into an interface (not exported) in `ActionsBar.tsx`.

Reply:
addressed in b07a19a 😋